config/index.js

```diff
@@ -493,6 +493,57 @@ var conf = convict({
       default: '.+@mozilla\\.com$',
       env: 'LASTACCESSTIME_UPDATES_EMAIL_ADDRESSES'
     }
+  },
+  signinUnblock: {
+    codeLength: {
+      doc: 'Number of base36 digits to make up an unblockCode',
+      default: 8,
+      env: 'SIGNIN_UNBLOCK_CODE_LENGTH'
+    },
+    codeLifetime: {
+      doc: 'How long an unblockCode should be valid for',
+      format: 'duration',
+      default: '1 hour',
+      env: 'SIGNIN_UNBLOCK_CODE_LIFETIME'
+    },
+    enabled: {
+      default: true
+    },
+    allowEmailRegex: {
+      doc: 'If feature enabled, allow sign-in unblock for email addresses matching this regex.',
+      format: Array,
+      default: [
+        '.+@mozilla\.com$'
+      ],
+      env: 'SIGNIN_UNBLOCK_ALLOW_EMAIL_REGEX'
+    },
+    forceEmailRegex: {
+      doc: 'If feature enabled, force sign-in unblock for email addresses matching this regex.',
+      format: Array,
+      default: [],
+      env: 'SIGNIN_UNBLOCK_FORCE_EMAIL_REGEX'
+    },
+    sampleRate: {
+      doc: 'signin unblock sample rate, between 0.0 and 1.0',
+      default: 1.0,
+      env: 'SIGNIN_UNBLOCK_RATE'
+    },
+    supportedClients: {
+      doc: 'support sign-in unblock for only these clients',
```

Comment:
Do we actually need/want this option for this feature? Unless I misunderstand (I probably do), the sign-in experience isn't materially changed by this feature on clients that don't support it. So having it enabled on those legacy clients won't do any harm?

I only ask because it would make the feature flag function simpler if we ditched it.